Update README.md - Support strategy info (#1391)
Adyen/adyen-magento2: README.md

* 2.3.7 and higher
* 2.4 and higher

## Support & Maintenance

There are three levels of support we provide for our Magento 2 plugins.
1. Level 1 > 12 months  - full support (including new features) *Keep in mind that some new features are not possible on older versions, therefore this is not inclusive of ALL new features that are built.* 
2. Level 2 > 12 months - 18 months - high priority bugs / security only 
3. Level 3 > 18 months - 24 months  - security updates only (no support from support team)
*** After Level 3 there is no support or maintenance from Adyen and it should be treated as a native integration of the merchant.

> The schedule for the current maintained branches of the Magento 2 plugin.

Reviewer: ```suggestion
The schedule for the current maintained branches is:
```

* 2.3.7 and higher
* 2.4 and higher

## Support & Maintenance

We provide three levels of support:
1. **Level 1 (12 months)**: full support (including new features) *Keep in mind that some new features are not possible on older versions, therefore this is not inclusive of ALL new features that are built.* 
2. **Level 2 (12 months - 18 months)**: high priority bugs / security only 

> 3. **Level 3 (18 months - 24 months)**: security updates only (no support from support team)

Reviewer 2: ```suggestion
3. **Level 3 (18 months - 24 months)**: security updates only (no support from support team).
```

* 2.3.7 and higher
* 2.4 and higher

## Releases

1. **Major** releases are done ONLY when we absolutely need to. We try to not to introduce breaking changes and do major releases as rare as possible. Average now is **yearly**.
2. **Minor** releases are scheduled but not limited to **once a month** (first week of a month). 

> 3. After each bug fix is resolved, we aim to make a **patch** release. A patch releas is scheduled but not limited to **once in 2 weeks.**

Reviewer 2: ```suggestion
3. After each bug fix is resolved, we aim to make a **patch** release. A patch release is scheduled but not limited to **once every 2 weeks.**
```

> **Note: This can be subject to change based on the load and dependancies of the Integration tools team. **

Reviewer 2: ```suggestion
**Note: This can be subject to change based on the load and dependancies of Adyen's Integration Tools team.**
```

## Support & Maintenance

> There are three levels of support we provide for our Magento 2 plugins.

Reviewer 2: ```suggestion
There are three levels of support we provide for our Magento 2 plugin.
```

1. Level 1 > 12 months  - full support (including new features) *Keep in mind that some new features are not possible on older versions, therefore this is not inclusive of ALL new features that are built.* 

> 2. Level 2 > 12 months - 18 months - high priority bugs / security only 

Reviewer 2: ```suggestion
2. Level 2 > 12 months - 18 months - high priority bugs / security only.
```

Reviewer: A formatting suggestion for the level names: 
```suggestion
2. **Level 2 (12 months - 18 months)**: high priority bugs / security only 
```

3. Level 3 > 18 months - 24 months  - security updates only (no support from support team)

> *** After Level 3 there is no support or maintenance from Adyen and it should be treated as a native integration of the merchant.

Reviewer 2: ```suggestion

*** After Level 3 there is no support or maintenance from Adyen and it should be treated as a native integration of the merchant.
```

Reviewer: ```suggestion
> After Level 3 there is **no support or maintenance from Adyen** and it should be treated as a native integration of the merchant.
```
